ceph/ceph_admin/apply.py

```diff
@@ -128,7 +129,7 @@ def apply(self: ServiceProtocol, config: Dict) -> None:
             raise OrchApplyServiceFailure(self.SERVICE_NAME)
 
         # out value is "Scheduled <service_name> update..."
-        service_name = out.split()[1]
+        service_name = re.search(r"Scheduled\s(.*)\supdate", out).group(1)
```

Comment:
Nice... this solves the issue with multi line output emitted when the `apply` command has `verbose` setting enabled.